src/current/v24.1/plpgsql.md

```diff
@@ -409,6 +459,114 @@ BEGIN
   END
 ~~~
 
+### Perform transaction control
```

Comment:
Would this be better titled as "Control transactions"?

Reply:
Yes!